web/source/dom/touchAliasElement.ts

```diff
@@ -30,26 +30,83 @@ namespace com.keyman.dom {
   // If the specified HTMLElement is either a TouchAliasElement or one of its children elements,
   // this method will return the root TouchAliasElement.
   export function findTouchAliasTarget(target: HTMLElement): TouchAliasElement {
+    // The scrollable container element for the before & after text spans & the caret.
+    // Not to be confused with the simulated scrollbar.
     let scroller: HTMLElement;
 
+    // Designed to 'breadcrumb' elements to assist in diagnosis of an evasive known issue.
+    let elementLogger = function() {
+      let elementDescriber = function(element: HTMLElement) {
+        if(element) {
+          let childIndex = undefined;
+          let parent = element.parentElement;
+          if(parent) {
+            for(let child = parent.firstChild, i = 0; child; child = child.nextSibling, i++) {
+              if(child == element) {
+                childIndex = i;
+                break;
+              }
+            }
+          }
+
+          // ID:  should never be specified for TouchAliasElement components
+          // Class:  specified for the root TouchAliasElement and for its text spans
+          // index:  useful for identifying elements with no class info
+          return `tag type ${element.tagName}, class '${element.classList}', index under parent: '${childIndex}', id '${element.id}'`;
```

Comment:
Is `` ` `` going to cause a problem on old versions of Chrome? https://caniuse.com/?search=backtick

Reply:
It won't.  The compiled version of this line, as seen from the unminified build product:

```javascript
// ID:  should never be specified for TouchAliasElement components
// Class:  specified for the root TouchAliasElement and for its text spans
// index:  useful for identifying elements with no class info
return "tag type " + element.tagName + ", class '" + element.classList + "', index under parent: '" + childIndex + "', id '" + element.id + "'";
```